go/ql/src/experimental/CWE-176/UnicodeBypassValidation.qhelp

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE qhelp PUBLIC "-//Semmle//qhelp//EN" "qhelp.dtd">
+<qhelp>
+    <overview>
+        <p>Security checks bypass due to a Unicode transformation</p>
+        <p>
+            If ever a unicode tranformation is performed after some security checks or logical
+            validation, the
+            latter could be bypassed due to a potential Unicode characters collision.
+            The validation of concern are any character escaping, any regex validation or any string
+            verification.
+        </p>
+        <img src="./vulnerability-flow.png" alt="Security checks bypassed" />
```

Comment:
No images in qhelp I'm afraid -- you'll have to describe it in words, or link to a reference that expounds greater detail.